src/core/reduce.rkt

```diff
@@ -123,7 +123,8 @@
        (cons (if (ormap (compose (curry = 0) car) dens) 'NAN (apply / (car num) (map car dens)))
              (append (cdr num)
                      (map negate-term (append-map cdr dens)))))]
-    [`(sqrt ,arg)
+    ;; Commented out due to unsoundness issues
+    #;[`(sqrt ,arg)
```

Comment:
Hmm, what's the unsoundness issue? On the one hand I feel like it's better to solve than remove, but on the other hand there's an argument for slowly stripping `reduce.rkt` of its functionality and leaving that to a simplify pass.

Reply:
It sometimes producing `sqrt(-1)` somehow

Reply:
Wait, to clarify—Taylor produces `sqrt(-1)`? Or the reducer produces `sqrt(-1)` when not given that as input? Or the reducer produces `sqrt(-1)` when given something like `sqrt(1 - 2)` as input? I think this code uses `eval-application` so it should only do the square root when legal. (But maybe there's something else at issue?)

Reply:
Hmm I remember seeing `sqrt(-1)` in the output of taylor before I added the check.
Let me try re-enabling this and seeing if our results stay good